src/injected/styles/injected.scss

```diff
@@ -389,27 +352,59 @@
         font-weight: $fontWeightSemiBold;
     }
 
-    .insights-dialog-main-override .insights-dialog-fix-instruction-container {
-        margin-bottom: 20px !important;
-        font-size: 14px;
-    }
+    .insights-dialog-main-override,
+    .insights-dialog-main-override-shadow {
+        .insights-dialog-title {
+            margin-bottom: 16px !important;
+            padding-bottom: 0px !important;
+            color: $primary-text !important;
+            font-size: 21px !important;
+            font-weight: $fontWeightSemiBold !important;
+            letter-spacing: -0.02em !important;
+        }
 
-    .insights-dialog-main-override .insights-fix-instruction-list {
-        padding-left: 5% !important;
-        margin: 0 !important;
-    }
+        .insights-dialog-section-title {
+            margin-top: 16px !important;
+            margin-bottom: 4px !important;
+            color: $primary-text !important;
+            font-size: 15px !important;
+            font-weight: $fontWeightSemiBold !important;
+        }
 
-    .insights-dialog-main-override .insights-fix-instruction-title {
-        font-weight: 600 !important;
-    }
+        .insights-dialog-content {
+            color: $secondary-text !important;
+            font-size: 14px !important;
 
-    .insights-dialog-main-override-shadow .insights-fix-instruction-title {
-        font-weight: 600 !important;
-        margin-top: 15px;
-    }
+            ul {
+                margin: 8px !important;
+                padding-left: 8px !important;
+            }
+        }
+
+        .insights-dialog-target-button-container {
+            margin-top: 16px !important;
+
+            button {
+                padding-left: 8px !important;
+                padding-right: 8px !important;
+            }
+
+            .ms-Button-label {
```

Comment:
Should these styles be '!important' as well?

Reply:
Yeah, probably. It feels disgusting to use `!important` like this but we probably should be doing so as long as we're outside the shadow dom and competing with page styling that might use it...